ideascube/templates/ideascube/includes/cards/deus-ex-silicium.html

```diff
@@ -1,3 +1,2 @@
 {% load i18n %}
-{% trans "learn" as theme %}
-{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme_class="learn" name="Deus Ex Silicium" description="Vidéos dédiées à la science des semi-conducteurs" theme=theme %}
+{% include 'ideascube/includes/cards/_kiwix.html' with card=card class="deus-ex-silicium" theme="learn" %}
```

Comment:
Yes, but I will not do it :)
Changes about the theme_class and the package+card management update the same lines in a lot of files and I will not go through all those files and rewrite a patch only for a better commit splitting. (Plus the _kiwix.html will change again in a next future to just be kiwix.html)